### ISSUE TITLE
Update Accessibility Report

### ISSUE THREAD

**Issue by drn05r** [eprints/eprints3.5#307]
EPrints now has a built in Accessibility report at /accessibility/report.html the testing of pages with the WAVE Accessibility tool should be carried out and the report should be updated just prior to the full release of 3.5.0.

**Comment by drn05r** [comment]
This is basically done.  Just waiting on final changes to be reviewed in pr #406
